spec/movements_spec.rb

```diff
@@ -0,0 +1,49 @@
+$LOAD_PATH << File.expand_path('. ./lib', __FILE__)
+# require 'martian_robots'
+require 'movements'
+
+RSpec.describe Movements do
+
+  let (:movements) {Movements.new}
+
+  it "returns E if robot's initial orientation is N" do
+    expect(movements.change_orientation_clockwise("N")).to eq("E")
+  end
+
+  it "returns W if robot's initial orientation is S" do
+    expect(movements.change_orientation_clockwise("S")).to eq("W")
+  end
+
+  it "returns S if robot's initial orientation is W" do
+    expect(movements.change_orientation_anticlockwise("W")).to eq("S")
+  end
+
+  it "returns N if robot's initial orientation is E" do
+    expect(movements.change_orientation_anticlockwise("E")).to eq("N")
+  end
+
+  it "returns 2 as value for coordinate x if robot starts from [1,1] and moves East" do
+    coordinates = [1,1]
+    movements.move_forward('E',coordinates)
+    expect(coordinates[0]).to eq(2)
+  end
+
+  it "returns 2 as value for coordinate y if robot starts from [1,1] and moves North" do
+    coordinates = [1,1]
+    movements.move_forward('N',coordinates)
+    expect(coordinates[1]).to eq(2)
```

Comment:
Nice work, I like that you are passing around coordinates and getting the Movements class to update them. 
One suggestion is to use a hash instead of an array; this would increase readability of your code. Your hash would look like this:
`coordinates = {x: 1, y: 2}`
And you would check y by simply looking it up like so: `coordinates[:y]`.
This makes your test more expressive: at the moment you have to know that `y` is the first element in the array, whereas with the hash you just ask for it.